### PR TITLE
[IN-2340] Update brew cask installs

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Review by bitrise-tom — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

File: roles/java/tasks/main.yml [hunk 1]
```diff
@@ -55,14 +56,14 @@
...
+  homebrew:
+    name: openjdk@8
     state: present
 
-- name: brew cask install adoptopenjdk11
-  homebrew_cask:
-    name: adoptopenjdk11
+- name: brew install openjdk11
+  homebrew:
+    name: openjdk@11
```
Comment (bitrise-tom) [inline roles/java/tasks/main.yml:66]: need to add the `install_options: cask` here

File: roles/java/tasks/main.yml [hunk 2]
```diff
@@ -55,14 +56,14 @@
     name: AdoptOpenJDK/openjdk
     state: present
 
-- name: brew cask install adoptopenjdk8
-  homebrew_cask:
-    name: adoptopenjdk/openjdk/adoptopenjdk8
+- name: brew install openjdk8
+  homebrew:
+    name: openjdk@8
     state: present
```
Comment (bitrise-tom) [inline roles/java/tasks/main.yml:62]: need to add the `install_options: cask` here

File: roles/java/tasks/main.yml [hunk 3]
```diff
@@ -34,9 +34,10 @@
   when: result.rc == 0
 
 - name: brew uninstall latest java
-  homebrew_cask:
+  homebrew:
     name: java
     state: absent
+    install_options: ignore-dependencies
```
Comment (bitrise-tom) [inline roles/java/tasks/main.yml:40]: need to add the install_options: cask here

**Pull request merged**